package.json

```diff
@@ -61,7 +61,7 @@
     "eslint-loader": "^1.9.0",
     "eventsource-polyfill": "^0.9.6",
     "express": "^4.15.3",
-    "flatpickr": "4.4.4",
```

Comment:
Existing any reason of have `flatpickr` as a `devDependencies`? I took `popper.js` as an example which is as a `dependencies` instead.

Reply:
Yes, it was because of this import: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-calendar/src/calendar.controller.js

But it's the same for Popper.js. I'll add it as a dependency.